app/controllers/photo-albums/photo-album/photos/index.js

```diff
@@ -8,15 +8,38 @@ export default Controller.extend(FileSaverMixin, {
   fetch: service(),
   photoSorting: ['-createdAt'],
   sortedPhotos: sort('model.photos', 'photoSorting'),
+  modalIsOpen: false,
+  downloadStarted: false,
+  downloadFinished: false,
+  modalTitle: 'Fotoalbum downloaden',
+  modalText: 'Het inpakken en downloaden van het album kan even duren. Terwijl de download bezig is kun je de webstek niet gebruiken en dit tabblad niet sluiten.',
   actions: {
     downloadAlbum() {
       let model = this.get('model');
+      this.set('downloadStarted', true);
+      this.set('modalTitle', 'Dit kan even duren...');
+      this.set('modalText', 'De download is gestart. Terwijl de download bezig is kun je de webstek niet gebruiken en dit tabblad niet sluiten.');
+
       return this.get('fetch').fetch(`/photo_albums/${model.id}/zip`).then(response => {
         return response.blob().then(blob => {
```

Comment:
Ik denk dat async/await gebruiken mooier is dan `.then`. Is ook mooie syntax om te kennen. Maar.... Niet echt onderdeel van deze PR misschien omdat je deze code niet aangeraakt hebt. 

Aan jou de keuze

Reply:
Ja, dat was de vorige pr, maar wat stel je precies voor?

Reply:
Het wordt dan iets als dit:

```javascript
async downloadAlbum() {
  ...
  const response = await this.get('fetch').fetch(...);
  const data = await response.blob();
  this.saveFileAs(/*...*/);
}
```

Het nadeel ervan is dat je geen error handling doet, maar dat was nu ook al niet het geval geloof ik.

Reply:
Ah, ik heb dat nooit zo gedaan, maar zal er naar kijken.